update_data.py

```diff
@@ -336,8 +359,7 @@ def process_request(request, campaign, cursor, table):
 
 conn.commit()
 
-
-# TWiki samples
+# TWiki samples - old mode
 with open('MainSamplesFall18.txt') as f:
```

Comment:
If you are using `update_twiki.py` then this can be removed

Reply:
Thanks! I think it's not ready yet for the change, so I will leave it as it is for now.